GitVersionCore/Configuration/Init/BuildServer/AppVeyorSetup.cs

```diff
@@ -0,0 +1,122 @@
+namespace GitVersion.Configuration.Init.BuildServer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using GitVersion.Configuration.Init.Wizard;
+    using GitVersion.Helpers;
+
+    class AppVeyorSetup : ConfigInitWizardStep
+    {
+        protected override StepResult HandleResult(string result, Queue<ConfigInitWizardStep> steps, Config config, string workingDirectory, IFileSystem fileSystem)
+        {
+            switch (result)
+            {
+                case "0":
+                    steps.Enqueue(new EditConfigStep());
+                    return StepResult.Ok();
+                case "1":
+                    GenerateBasicConfig(workingDirectory, fileSystem);
+                    steps.Enqueue(new EditConfigStep());
+                    return StepResult.Ok();
+                case "2":
+                    GenerateNuGetConfig(workingDirectory, fileSystem);
+                    steps.Enqueue(new EditConfigStep());
+                    return StepResult.Ok();
+            }
+            return StepResult.InvalidResponseSelected();
+        }
+
+        void GenerateBasicConfig(string workingDirectory, IFileSystem fileSystem)
+        {
+            WriteConfig(workingDirectory, fileSystem, @"install:
+  - choco install gitversion.portable -pre
```

Comment:
Just to check... Are you aware that on the latest version of Chocolatey, this is no longer a silent operation, the user will be prompted whether they want to continue with the installation or not.  As a result, we might want to include the ```-y``` parameter, so automatically proceed with the installation.